images/oc-build-deploy-dind/openshift-templates/mongo/template.yml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: amazeeio-openshift-template-mongo
+parameters:
```

Comment:
these seem not up to date, see https://github.com/amazeeio/lagoon/blob/master/images/oc-build-deploy-dind/openshift-templates/mariadb/template.yml#L6 as an example